lib/phoenix_live_view/router.ex

```diff
@@ -103,14 +118,61 @@ defmodule Phoenix.LiveView.Router do
 
   """
   defmacro live(path, live_view, action \\ nil, opts \\ []) do
+    vsn = session_vsn()
```

Comment:
We should invoke this just once, likely on `use Phoenix.LiveView.Router`, otherwise, each default route is going to have a different version, which means you will never be able to live redirect between them.